### ISSUE TITLE
PHP 8.1 support

### ISSUE THREAD

**Issue by crpmax** [RikudouSage/QrPaymentCZ#47]
Hello,
please add PHP 8.1 support.

With PHP 8.1 it throws `strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated`
in `QrPayment.php` on line `195`.

Thanks.

**Comment by RikudouSage** [comment]
Which version are you using?

**Comment by crpmax** [comment]
I am using PHP 8.1.3
But the deprecation comes from PHP 8.1
look here: https://www.php.net/manual/en/migration81.deprecated.php

**Comment by RikudouSage** [comment]
Yeah, sorry, I meant the version of my library, as far as I know the error shouldn't be triggered in the newest version.

**Comment by crpmax** [comment]
I see the problem.
I ran composer update before, so I thought I had the latest version, but it was v4.2.2 - the latest v4.
But for version 5 I see I have to rewrite the code a bit...

Thank you anyway and sorry for my error.

**Comment by RikudouSage** [comment]
Glad I could help! Let me know if it works after you update to 5.x, I'll leave the issue open for now.
